src/lab_orchestrator_lib_auth/auth.py

```diff
@@ -55,29 +58,28 @@ def decode_auth_token(token: str, secret_key: str) -> Optional[LabInstanceTokenP
     :param secret_key: Key that should be used to decrypt the token.
     :return: The data that is contained in the token or None if decode goes wrong.
     """
-    try:
-        data = jwt.decode(token, secret_key, algorithms=['HS256'])
-        return LabInstanceTokenParams(
-            lab_id=data['lab_instance']['lab_id'],
-            lab_instance_id=data['lab_instance']['lab_instance_id'],
-            namespace_name=data['lab_instance']['namespace_name'],
-            allowed_vmi_names=data['lab_instance']['allowed_vmi_names']
-        )
-    except:
-        return None
-
-
-def verify_auth_token(token: str, vmi_name: str, secret_key: str) -> Optional[LabInstanceTokenParams]:
+    data = jwt.decode(token, secret_key, algorithms=['HS256'])
```

Comment:
without the catch statement you need to add `:raise:` to the documentation string and change the return type hint to `LabInstanceTokenParams`

Reply:
explicitly listing all of them or like this:

    :raises: Any errors occuring while validation
?

Reply:
okay so they are explicitly listed now.